build_web_content.py

```diff
@@ -89,5 +102,8 @@
 subprocess.run(
     ["sed", "-i", "s/_gz\[\]/_gz\[\] PROGMEM/g", targetpath + "/webcontent.h"]
 )
-
+# convert all undigned to const unsigned...
+subprocess.run(
```

Comment:
I'm getting this error when I build:

sed: 1: "src/www/build/webcontent.h": unterminated substitute pattern
sed: 1: "src/www/build/webcontent.h": unterminated substitute pattern

Reply:
I built it on my Mac.  platformio craters on my ubuntu system and I haven't spent any time looking into why.

Reply:
Okay, I will see if I can reproduce this problem on my Mac... maybe end of week as I am traveling for business this week.

Reply:
Just need to add "-e" to the sed command

    ["sed", "-ie", "s/_gz\[\]/_gz\[\] PROGMEM/g", targetpath + "/webcontent.h"]
    ["sed", "-ie", "s/unsigned/const unsigned/g", targetpath + "/webcontent.h"]

Reply:
Works on both Ubuntu 22.04 and macOS with the -e option